manim/mobject/svg/text_mobject.py

```diff
@@ -822,6 +823,22 @@ def gen_chars(self):
                 submobjects_char_index += 1
         return chars
 
+    def gen_words(self):
```

Comment:
I _think_ this should be a private method (in python, you can conventionnally mark a method as private by putting `_` in front of the name of the method, like this 
```suggestion
    def _gen_words(self):
```
(don't apply the suggestion, as there is a call of the method above)